### ISSUE TITLE
Create separate repository for grafana dashboards

### ISSUE THREAD

**Issue by ChrisKujawa** [camunda/camunda#13481]
**Description**

Right now we store our dashboards under https://github.com/camunda/zeebe/tree/main/monitor/grafana and other dashboards under https://github.com/camunda/zeebe/tree/main/monitor/grafana/dashboards

These dashboards are scraped from our Grafana installations (in our benchmarks cluster but also from saas)

It would make sense to move them to a separate repository dedicated to the Grafana dashboards. 

Pro:

 a) make it easier to find all of them, since the repository just exists for that purpose (For example I was not aware that we store also dashboards from other components)
 b) we would able to make easier changes without relying on bors (and waiting on the whole pipeline) etc. 

Con:

 a) we might miss updates (?) (not sure whether this is true)
 b) is it easier to miss when not aware, that it is in a different repo (we could link to it)
 c) potential breaking change, due to the move (but we could add a readme with link to it)

**Comment by npepinpe** [comment]
> we would able to make easier changes without relying on bors (and waiting on the whole pipeline) etc.

:fire:

**Comment by remcowesterhoud** [comment]
ZPA Triage:

- Makes sense to move them to a separate repo to us.
- It is breaking, we must make sure to update [this](https://github.com/camunda-cloud/grafana-dashboards/blob/main/dashboards/operate/operate-dashboard.yaml) page for example. However, it is not part of the public API so we do not guarantee backwards compatibility over these dashboards.
- We would propose to copy them to a new repo first, and leave them in the Zeebe repo to be backwards compatible
- We should write down that the ones in the Zeebe repo are deprecated, no longer updated and instruct users to update to the new repository.
- After a certain amount of time we can remove them from the Zeebe repo.
- According to the[ Zeebe Status-Quo document](https://docs.google.com/document/d/1v9GPzzdrycvPHpD66GHigdQBMQ4WeLcr2UPDvU73620/edit#heading=h.x81xdbj9oihr) Grafana is a responsibility of the ZDP team. We believe Grafana is more of a shared responsibility and thus we won't remove this issue from our board.

**Comment by megglos** [comment]
ZDP-Triage:
- consider it a shared responsibility as well
- marking it as later as there is no urgency right now

**Comment by ChrisKujawa** [comment]
@multani i think we can close this with your epic

**Comment by marcosgvieira** [comment]
🔄 **Status updated automatically (Core Features)**

**From:** Inbox  
**To:** Ready  

**Reason:** The issue already has an assignee and a parent issue.
